README.MD

```diff
@@ -44,7 +44,7 @@ await foreach (var tool in client.ListToolsAsync())
 // Execute a tool (this would normally be driven by LLM tool invocations).
 var result = await client.CallToolAsync(
     "echo",
-    new() { ["message"] = "Hello MCP!" },
+    new Dictionary<string, object?>() { ["message"] = "Hello MCP!" },
```

Comment:
Has the explicit type been added for emphasis, or because it is now necessary?
```suggestion
    new Dictionary<string, object?> { ["message"] = "Hello MCP!" },
```